pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go

```diff
@@ -9,14 +9,15 @@ import (
 // NetworkAddonsConfigSpec defines the desired state of NetworkAddonsConfig
 // +k8s:openapi-gen=true
 type NetworkAddonsConfigSpec struct {
-	Multus           *Multus           `json:"multus,omitempty"`
-	LinuxBridge      *LinuxBridge      `json:"linuxBridge,omitempty"`
-	Ovs              *Ovs              `json:"ovs,omitempty"`
-	KubeMacPool      *KubeMacPool      `json:"kubeMacPool,omitempty"`
-	ImagePullPolicy  corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
-	NMState          *NMState          `json:"nmstate,omitempty"`
-	MacvtapCni       *MacvtapCni       `json:"macvtap,omitempty"`
+	Multus                *Multus                `json:"multus,omitempty"`
+	LinuxBridge           *LinuxBridge           `json:"linuxBridge,omitempty"`
+	Ovs                   *Ovs                   `json:"ovs,omitempty"`
+	KubeMacPool           *KubeMacPool           `json:"kubeMacPool,omitempty"`
+	ImagePullPolicy       corev1.PullPolicy      `json:"imagePullPolicy,omitempty"`
+	NMState               *NMState               `json:"nmstate,omitempty"`
+	MacvtapCni            *MacvtapCni            `json:"macvtap,omitempty"`
 	SelfSignConfiguration *SelfSignConfiguration `json:"selfSignConfiguration,omitempty"`
+	HyperConvergedConfig  *HyperConvergedConfig  `json:"hyperConvergedConfig,omitempty"`
```

Comment:
HCO depends on us, not the other way around. No need to mention them.

Calling it `Placement` would be a better fit IMO.

Reply:
Renamed to PlacementConfiguration, which seems to fit a bit better next to `SelfSignConfiguration`, are you okay with this?

Reply:
Sounds good, thanks